ui_framework/components/tabs/tabs.test.js

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+import { render, shallow } from 'enzyme';
+import { requiredProps } from '../../test/required_props';
+import sinon from 'sinon';
+
+import {
+  KuiTabs,
+} from './tabs';
+
+import {
+  KuiTab,
+} from './tab';
+
+describe('KuiTabs', () => {
+  const tabs = [
+    'Cobalt',
+    'Dextrose',
+    'Helium-3',
+    'Monosodium Glutamate',
+  ];
+
+  describe('throws an error', () => {
```

Comment:
I feel like these tests should just test the `selectedTabIndexCheck` function itself without being attached to a component.